docs/logging-cpython-picologging-comparison.md

```
@@ -313,11 +313,14 @@ applications.
 framework with flexible configuration and extensibility. Microsoft’s
 `picologging` reimplements the same API in (mostly) C++ for high performance.
-Key differences include: **class implementation** (Python classes vs C structs),
-**locking** (global RLock plus per-handler RLock vs per-handler C++ mutex
-with no global lock), **data structures** (dynamic vs static fields), and
-**features supported** (full logging API vs a subset). These design choices give
-picologging much higher throughput at the cost of some flexibility.
+
+Key differences include:
+
+- **class implementation** (Python classes vs C structs)
+- **locking** (global RLock plus per-handler RLock vs per-handler C++ mutex, 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)